lib/ronin/cli/commands/ip.rb

```diff
@@ -236,10 +240,20 @@ def process_value(ip)
         #   The formatted IP address.
         #
         def format_ip(ip)
+          if ip.ipv4?
+            format_ipv4(ip)
+          else
+            format_ipv6(ip)
+          end
+        end
+
+        private
+
+        def format_ipv4(ip)
```

Comment:
Might want to add documentation to these methods even though they are private, in case someone else looks at them. Documentation can be copy/pasted from `format_ip`.